### PR TITLE
feat: make duration_seconds optional

### PR DESCRIPTION
**Pull request by joseph-sentry**
also adds a test for a JUnit XML that only has a single empty `testsuites` tag

### CODE REVIEW

**Review by Swatinem — APPROVED** [review]

File: src/junit.rs [hunk 1]
```diff
@@ -63,8 +63,7 @@ fn populate(
         .time
         .as_deref()
         .or(testsuite_time)
-        .ok_or_else(|| ParserError::new_err("No time/duration found"))?
-        .parse()?;
+        .map(|t| t.parse().unwrap());
```
Comment (Swatinem) [inline src/junit.rs:66]: as a followup, it would be good to not `unwrap` here.
we are dealing with untrusted user input, and crashing on not properly formatted numbers would be bad

**Review by Swatinem — APPROVED** [review]
